src/Propel/Runtime/ActiveQuery/QueryExecutor/AbstractQueryExecutor.php

```diff
@@ -78,7 +78,7 @@ public function __construct(Criteria $criteria, ?ConnectionInterface $con = null
      *
      * @return \Propel\Runtime\Connection\ConnectionInterface
      */
-    protected function getConnection(ServiceContainerInterface $sc, string $dbName, bool $getWritableConnection = true): ConnectionInterface
+    private function retrieveConnection(ServiceContainerInterface $sc, string $dbName, bool $getWritableConnection = true): ConnectionInterface
```

Comment:
Can we keep protected and not use private in this context?

Reply:
Right, forgot about that, sorry